admin/actions.php

```diff
@@ -685,6 +685,16 @@ function hmbkp_ajax_cron_test() {
 
 	}
 
+	// Skip test if Cron is disabled
+	if ( defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON ) {
+
+		delete_option( 'hmbkp_wp_cron_test_failed' );
+
+		echo 1;
+
+		die;
+	}
+
 	// Skip the test if they are using Alternate Cron
 	if ( defined( 'ALTERNATE_WP_CRON' ) ) {
```

Comment:
We should also check if `ALTERNATE_WP_CRON` is `true` like you're doing for `DISABLE_WP_CRON`